src/pages/patientView/PatientViewPage.tsx

```diff
@@ -544,6 +558,68 @@ export class PatientViewPageInner extends React.Component<
         return this.patientViewPageStore;
     }
 
+    @action.bound
+    private handleReturnToStudyView() {
+        const patientIdentifiers = this.pageStore.patientIdsInCohort.map(p => {
+            const patientIdParts = p.split(':');
+            return {
+                patientId: patientIdParts[1],
+                studyId: patientIdParts[0],
+            };
+        });
+        const queriedStudies: Set<string> = new Set(
+            patientIdentifiers.map(p => p.studyId)
+        );
+
+        const studyPage = window.open(
```

Comment:
lets put a comment explaining this little hacky mechanism for "posting" data to the new window.  
"... we need to do this because url method would be too long"